### DISCUSSION TITLE
Free Steering

### DISCUSSION THREAD

**Discussion by Jayshua83** [Mogli12/KeyboardSteer#756] [General]
I don't think Free Steering is working.

Free Steering is OFF and Adaptive steering is ON and the steering does not rotate back.

MP / Dedi Server

**Comment by Mogli12** [comment]
I do not have this problem.

**Comment by Jayshua83** [comment]
So that's it? Wow... ok... :|

**Comment by Mogli12** [comment]
Sorry, but what do you expect?

After reading this short issue I started the game, entered a vehicle and tapped on key A a bit. I drove futher and the vehicle steered back.

Please read section Developer Version of README.md.
